reinforcement_learning/rl_cartpole_ray/rl_cartpole_ray_gymEnv.ipynb

```diff
@@ -9,7 +9,7 @@
     "---\n",
```

Comment:
<p>Any reason to expose this parameter as compared to the previous version?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1520/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
SageMaker Debugger is a new feature released in last re:Invent. Prior to that the summary data collected by `tf.summary` didn't interact with SageMaker.